apps/cms.c

```diff
@@ -940,6 +944,27 @@ int cms_main(int argc, char **argv)
             si = CMS_add1_signer(cms, signer, key, sign_md, tflags);
             if (si == NULL)
                 goto end;
+            if (flags & CMS_CADES) {
```

Comment:
While digging about the way CAdES signature should be verified, I found that this CMS_CADES flag value should not be done here .
this processing should be moved into libcrypto code instead.
More precisely, inside the ` if (!(flags & CMS_NOATTR)) {` block : 
https://github.com/openssl/openssl/blob/08ddd3023062d26f707f6a7422aecc8612098ec1/crypto/cms/cms_sd.c#L334

Because of the possible interaction with some others flag.
And with this change, any people wanting to use CAdES, just have to changes the `flags` in its own code.